File: apps/backend/src/main/java/com/qrancae/app/model/MaintenanceTask.java
package com.qrancae.app.model;

import lombok.Data;

@Data
public class MaintenanceTask {

    private String maintCable;
    private String maintQr;
    private String maintPower;
    private String status;
    private String maintDate;  // LocalDateTime에서 String으로 변경
    private String alarmDate;  // LocalDateTime에서 String으로 변경
    private String alarmMsg;
    private String sRackNumber;
    private String sRackLocation;
    private String cableIdx;
}

File: apps/mobile/app/src/main/java/com/qrancae/mobile/model/MaintenanceTask.kt
package com.qrancae.mobile.model

import com.google.gson.annotations.SerializedName

data class MaintenanceTask(
    val maintCable: String?,
    val maintQr: String?,
    val maintPower: String?,
    @SerializedName("status") val status: String,
    @SerializedName("maintDate") val maintDate: String?,
    @SerializedName("alarmDate") val alarmDate: String?,
    val alarmMsg: String?,
    @SerializedName("srackNumber") val sRackNumber: String?,  // JSON 필드와 매핑
    @SerializedName("srackLocation") val sRackLocation: String?,  // JSON 필드와 매핑
    val cableIdx: String?
)
